docs/index.njk

```diff
@@ -113,7 +113,7 @@ title: Home
               <span class="opacity-50 icon fas fa-layer-group me-1" aria-hidden="true"></span>Foundation
             </h3>
             <p class="opacity-75">Pelican is built on small parts working together. These parts are explained so all projects can build upon Pelican successfully.</p>
-            <a href="/foundation/" class="btn btn-accent w-100 shadow-none">
+            <a href="/foundation/" class="btn btn-accent w-100">
               Learn how it works<span class="opacity-50 icon fas fa-arrow-right ms-1" aria-hidden="true"></span>
             </a>
```

Comment:
A few accent buttons in the `footer-utility.njk` partial still have shadow-none classes. Should those be removed as well?

Reply:
Fixed